projects/plugins/jetpack/extensions/shared/memberships/subscribers-affirmation.js

```diff
@@ -289,9 +290,22 @@ function SubscribersAffirmation( { accessLevel, prePublish = false } ) {
 		<p>
 			{ createInterpolateElement( text, {
 				strong: <strong />,
+				link: <a href={ getJetpackEmailStatsLink( blogId, postId ) } />,
 			} ) }
 		</p>
 	);
 }
 
+/**
+ * Get the Jetpack email stats link for the given post ID.
+ *
+ * @param {number} blogId - The ID of the blog.
+ * @param {number} postId - The ID of the post.
+ *
+ * @return {string} - The Jetpack email stats link for the given post.
+ */
+function getJetpackEmailStatsLink( blogId, postId ) {
+	return `/wp-admin/admin.php?page=stats#!/stats/email/opens/day/${ postId }/${ blogId }`;
```

Comment:
This won't work on sites where [WordPress is installed in a subdirectory](https://developer.wordpress.org/advanced-administration/server/wordpress-in-directory/). Since `getSiteData` gives us `admin_url`, I would recommend using that to build the URL.

Reply:
Fixed in [c3d340e](https://github.com/Automattic/jetpack/pull/43193/commits/c3d340efca62906b0e3cf0891f33f13e96a4f4ac)